src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -531,6 +531,9 @@ bool emberAfOperationalCredentialsClusterAddNOCCallback(app::CommandHandler * co
     FabricIndex fabricIndex = 0;
     Credentials::GroupDataProvider::KeySet keyset;
 
+    uint8_t compressed_fabric_id_buffer[sizeof(uint64_t)];
```

Comment:
Oh, this is the same fix at https://github.com/project-chip/connectedhomeip/pull/15931

Reply:
Yes, just so I could pass CI.  I'll rebase on master to get rid of this part now that that merged.